index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/style.css"/>
+    <title>Pete Heyes - Developer Chap</title>
+    <link href="https://fonts.googleapis.com/css?family=Yellowtail" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+</head>
+<body>
+    <div class="split1">
+        <nav>
+            <ul class="nav-items">
+                <a href="#home"><li>Home</li></a>
```

Comment:
Block level Element within Inline Element, should be Inline within Block <li><a></a></li>